documentation/modules/ova-prerequisites.adoc

```diff
@@ -4,23 +4,28 @@
 
 :_content-type: REFERENCE
 [id="ova-prerequisites_{context}"]
-= Open Virtual Application (OVA) prerequisites
+= Open Virtual Appliance (OVA) prerequisites
 
-The following prerequisites apply to Open Virtual Application (OVA) file migrations:
+The following prerequisites apply to Open Virtual Appliance (OVA) file migrations:
 
-* All OVA files are created by or compatible with VMware vSphere.
-* The OVA files are in a shared NFS share that has the following security settings: `rw`.
-* Each OVA file is in a separate folder in the NFS share -  sub-folders are not checked.
```

Comment:
we don't need separate folders but I think we should mention that we require at least one folder under the NFS share that holds the files and not directly on it.